src/polyswarm_api/api.py

```diff
@@ -349,6 +349,18 @@ def tag_link_update(self, sha256, tags=None, families=None, remove=False):
         """
         return self.generator.update_tag_link(sha256, tags=tags, families=families, remove=remove).execute().result
 
+    def tag_link_list(self, tags=None, families=None, or_tags=None, or_families=None):
+        """
+        Fetch all existing TagLinks for the provided tags.
+        :param tags: A list of tags that must be associated with the TagLinks listed.
+        :param families: A list of families that must be associated with the TagLinks listed.
+        :param or_tags: A list of tags that the TagLinks must be associated with at least one.
+        :param or_families: A list of families that the TagLinks must be associated with at least one.
```

Comment:
change to "A list of families where the ..."

Reply:
Similarly here.

Reply:
same here